### ISSUE TITLE
Accessibility 9.1.3.1a - remove h1 from header

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#309]
# Bug

Aktuell kann zwar per Sprung zur nächsten Landmark der Header auch übersprungen werden, sobald aber eine H1 angesprungen wird – was in der Regel der direkte Sprung zum Textinhalt der Seite ist – werden Nutzende wieder in den Header zurückgeführt. Das ist der Orientierung nicht wirklich zuträglich und es ist zu empfehlen, das noch einmal anzupassen. Die Erwartungen an den Aufbau einer Webseite ist, dass mit der H1 direkt in den Hauptbereich nach dem Header gesprungen werden kann.

## SOLL-Zustand

Der Title der Seite im Header sollte nicht als H1 ausgezeichnet sein, stattdessen sollte die H1 für die Überschrift der jeweiligen Seite genutzt werden.

--------------

# Bug

Currently, the header can also be skipped by jumping to the next landmark, but as soon as an H1 is jumped to - which is usually the direct jump to the text content of the page - users are taken back to the header. This is not really conducive to orientation and it is advisable to adjust this again. The expectation for the structure of a website is that the H1 can be used to jump directly to the main area after the header.

## Desired state

The title of the page in the header should not be marked as H1, instead the H1 should be used for the heading of the respective page.
